packages/now-cli/src/util/dev/server.ts

```diff
@@ -869,13 +869,6 @@ export default class DevServer {
     }
   }
 
-  shouldRebuild(req: http.IncomingMessage): boolean {
-    return (
-      req.headers.pragma === 'no-cache' ||
-      req.headers['cache-control'] === 'no-cache'
```

Comment:
Overall I think this is good. But it would be nice to trigger a rebuild without CTRL + C and now dev again. Is there still a solution for this? I guess just modify the source file?

Reply:
I guess modifying the source to trigger a rebuild is fine, since I don't think Next.js etc. do it differently.